Documentation/policy/language.rst

```diff
@@ -459,7 +459,7 @@ or are not know a priori, or when DNS is more convenient. To enforce policy on
 DNS requests themselves, see `Layer 7 Examples`_.
 
 IP information is captured from DNS responses per-Endpoint via a `DNS Proxy`_
-or `DNS Polling`_. An L3 `CIDR based`_ rule is generated for every ``toFQDNs``
+. An L3 `CIDR based`_ rule is generated for every ``toFQDNs``
```

Comment:
Please move that dot (`.`) to the previous line. Otherwise there will be a whitespace shown between `DNS Proxy` and the dot. You can see it here: https://github.com/cilium/cilium/blob/d7a27239fe3d42854c18baf751016d7955dc5d6e/Documentation/policy/language.rst

Reply:
Ok will fix it.